src/gafaelfawr/operator/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Kubernetes operator framework.
+
+This module imports all of the handlers for Gafaelfawr's Kubernetes operator
+and serves as an entry point for Kopf_.
+
+Examples
+--------
+Start the Gafaelfawr Kubernetes operator with the following command:
+
+.. code-block: shell
```

Comment:
Interesting. The module docstring is getting cut off here. https://gafaelfawr.lsst.io/v/DM-36458/dev/internals.html#module-gafaelfawr.operator I didn't expect that to happen.

I guess arguably that documentation could go in the user guide for gafaelfawr rather than the docstring.

Reply:
I only had one colon, so it thought this was a comment. That said, moving this is a good idea, since no one will find it there. I added it to the CLI page in the user-guide instead.